CHANGES

```diff
@@ -3,6 +3,7 @@
 - Fix failure when system properties modified when loading configuration
 - Update Jackson to 2.17.1
 - Update Jersey to 3.1.7
+- Add support for Enum array injection into configuration classes
```

Comment:
The code looks like supporting not only enum arrays. Am i reading it wrong?

Reply:
You are right, you can also inject arrays of any other supported types